react-validated-input-component/src/validated-input.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+class ValidatedInput extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: ''
+    };
+    this.passwordErrors = {};
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({ value: event.target.value });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+  }
+
+  checkPassword() {
+    this.passwordErrors =
```

Comment:
While setting the property in the method will work it would be better to use a return here